### PR TITLE
Added product update for Feb 23

### PR DESCRIPTION
**Pull request by sujaya-sys**

### CODE REVIEW

File: docs/change-log.md [hunk 1]
```diff
@@ -2,6 +2,31 @@
 
 This document provides an overview of the changes we are making over time. Feel free to reach out to us in case of any specific questions.
 
+## Product Update February 23rd, 2024
```
Comment (mathieu-benoit) [inline docs/change-log.md:5]: Could we add something around:
```
- **Improved:** Various minor UI enhancements have been implemented, including:
  - Count in lists
  - Others?
```

File: docs/change-log.md [hunk 2]
```diff
@@ -2,6 +2,31 @@
...
+
+- **New:** Introduced the [Delete Active Resources API](https://api-docs.humanitec.com/#tag/ActiveResource/operation/deleteActiveResource), allowing to detach an active resource. A detached resource remains intact but is no longer connected to the Platform Orchestrator.
+
+- **Fixed:** Switching frequently between Organizations no longer causes loading issues within an Application.
+
+#### Versioned component changes:
```
Comment (mathieu-benoit) [inline docs/change-log.md:11]: Could we add please?
- `humctl` 0.13.5 - https://github.com/humanitec/cli/releases/tag/v0.13.5
- `score-compose` 0.9.0 -https://github.com/score-spec/score-compose/releases/tag/0.9.0

**Review by chrishumanitec — APPROVED** [review]
LGTM

**Review by mathieu-benoit — APPROVED** [review]
LGTM

**Pull request merged**